data/avgi/avgi.txt

```diff
@@ -0,0 +1,1404 @@
+		"scandium"
+		"sheet metal"
+		"silica aerogel"
+		"silicon bronze"
+		"silver ingots"
+		"silver pipes"
+		"silver solder"
+		"silver wire"
+		"silver"
+		"sintered bronze"
+		"sodium metal"
+		"spring steel"
+		"stainless steel pipes"
+		"stainless steel"
+		"steel alloys"
+		"steel cable"
+		"steel gears"
+		"steel girders"
+		"steel plating"
+		"steel rebar"
+		"steel springs"
+		"steel wire"
+		"steel"
+		"superrefractory ceramics"
+		"tin wire"
+		"tin"
+		"titanium alloys"
+		"titanium"
+		"tool steel"
+		"tungsten boride"
+		"tungsten carbide"
+		"ultra high temperature superconductors"
+		"vanadium"
+		"yttrium aluminum garnet"
+		"zinc"
+		"zirconia plates"
+		"zirconia rods"
+		"zirconia"
+		"zirconium alloys"
+		"zirconium ceramics"
+		"zirconium diboride"
+		"zirconium hydride"
+		"zirconium hydrides"
+		"zirconium uranium carbide"
+		"zirconium"
+
+	commodity "Avgi Equipment"
+		"atomic circuits"
+		"atomic clocks"
+		"betatrons"
+		"bioreactors"
+		"breeder reactors"
+		"centrifuges"
+		"compact smeltries"
+		"cyclotrons"
+		"fission fragment etchers"
+		"free electron lasers"
+		"graphene fractal capacitors"
+		"graphene weavers"
+		"hydrolyzers"
+		"induction furnaces"
+		"industrial centrifuges"
+		"industrial lasers"
+		"laser cutters"
+		"lathes"
+		"linacs"
+		"mechanosynthesizers"
+		"molecular assemblers"
+		"molten oxide regolith smelters"
+		"molten salt reactors"
+		"nanobars"
+		"nanofabricators"
+		"nanofilament weavers"
+		"nanoflywheels"
+		"nanomesh"
+		"nanomotors"
+		"nanotensors"
+		"nanotube weavers"
+		"nuclear centrifuges"
+		"nuclear fuel reprocessors"
+		"nuclear furnaces"
+		"nuclear isomer batteries"
+		"nuclear plasma furnaces"
+		"particle bed furnaces"
+		"particle bed reactors"
+		"pebble bed reactors"
+		"plasma gasification separators"
+		"refractory electrodes"
+		"respirocyte energizers"
+		"rhodotrons"
+		"semiconductor fabs"
+		"solid oxide fuel cells"
+		"superconducting batteries"
+		"superconducting inductors"
+		"superconducting loops"
+		"superconducting magnetic energy storages"
+		"superconducting magnets"
+		"superconducting rings"
+		"synchrocyclotrons"
+		"synchrotron coils"
+		"synchrotrons"
+		"ultracapacitors"
+		"voltage multipliers"
+
+	commodity "Avgi Megaconstruction"
+		"boron nanofibers"
+		"boron nanofilaments"
+		"carbon nanofibers"
+		"carbon nanofilaments"
+		"diamond plates"
+		"diamond rods"
+		"diamond sheets"
+		"diamondoids"
+		"diamond nacre"
+		"graphene plates"
+		"graphene sheets"
+		"graphene tubes"
+		"graphene"
+
+	commodity "Avgi Nuclear Weapons"
+		"atomic propulsion packets"
+		"fission pills"
+		"nuclear mining charges"
+		"pulse drive units"
+		"shaped nuclear charges"
+
+	commodity "Avgi Radioisotopes"
+		"americium 242m"
+		"americium ceramics"
+		"americium pebbles"
+		"americium pellets"
+		"americium slugs"
+		"americium"
+		"californium ceramics"
+		"californium pebbles"
+		"californium pellets"
+		"californium slugs"
+		"californium"
```

Comment:
We typically avoid using anything that references real-life cities/lands/locations. For example, the Gegno Epidote was originally named the Muscovite, but it was changed because we wanted to avoid having a reference to Moscow. Though it's kind of a general agreement, @Amazinite is the one who said to change it, so I'd like his feedback here.

Reply:
>Those appear to be element names

They are, but again, the idea was that they were named after certain cities or countries, and having that be party of an alien faction seems pretty odd. It'd make more sense if these were special new names or something.

Reply:
Having something named after a planet is notably different than naming it after a modern country; Uranus isn't a forged group of people under a banner, it's more mythological and specifically references the stars. Americium and Californium are just, well, named after those places.

Reply:
With the example of the Muscovite - with the Gegno, it's. "alien ships named after general rocks", and Muscovite was a problem because it was a rock specifically named with Moscow in mind. The overall rock names, or planet names in thise case, is more generalized (and yes more in english), but Muscovite specifically references a city or place of a specific people in real life, which we want to avoid, as that's not directly something we represent in game.

Reply:
Greek Mythology doesn't really equate to naming something after a country though, a lot of the cosmos was named in these older times, naming something after a modern country is much less appropriate for something in the stars. Either way, I'd still like Amazinite to weigh in on it, I'm not specifically requesting changes, just saying it has been a case in the past and changed with the above reasoning.

Reply:
Fair enough.